spec/openapi.yaml

```diff
@@ -5226,14 +5201,7 @@ components:
             - Account1
             - Account2
         status:
-          description: >-
-            An array of message statuses
-          type: array
-          items:
-            type: object
-          example:
-            - FAILED
-            - DELIVERED
+          $ref: '#/components/schemas/MessageStatus'
```

Comment:
As in:
https://github.com/messagemedia/documentation/pull/33#discussion_r1006377349